### ISSUE TITLE
Clean up defunct/unmaintained web presences

### ISSUE THREAD

**Issue by ischoegl** [Cantera/enhancements#70]
**Abstract**

While main Cantera web presences are well maintained, there are several instances of defunct pages:

* [github.io website](https://cantera.github.io/docs/sphinx/html/index.html)
* [sourceforge repo](https://sourceforge.net/projects/cantera/)

that are 'frozen' at version 2.3. Simple clarifications / web redirects would ensure that users are guided to the most up-to-date content.

**Motivation**

Web searches bring up defunct / obsolete content.

**Possible Solutions**

Either create redirects or add disclaimers pointing users to the most up-to-date content.

**Other comments**

It may be good to add [CERFACS cantera](http://www.cerfacs.fr/cantera/description.php) to [affiliated packages](https://cantera.org/affiliated-packages.html) to clarify what the relationship is. It appears to be a fork based on the old sourceforge repo? The main reason I mention this is that some of the added mechanisms come up occasionally on the UG.

**PS:** Gitter looks like it’s no longer used.

**Comment by ischoegl** [comment]
@speth ... I guess this could go to discussions also as it doesn’t affect the actual code base (and it appears that Gitter is gone).

**Comment by speth** [comment]
I don't think there's much to discuss - I agree with all of your suggestions, they're just things that need to actually  be done.

I just deleted the defunct `Cantera/docs` repository which was the source of the `github.io` site. Hopefully that will disappear automatically now.

**Comment by ischoegl** [comment]
:+1: github.io is indeed gone. Sourceforge still shows recent download activity, which makes me wonder ...

**Comment by speth** [comment]
I don't want to delete the SourceForge site since I think it has some value as a historical archive, but I was able to make some updates there that should direct people to more current resources.

SourceForge had an option to indicate that the project has moved, which I pointed to the main repository here. That creates creates a blue badge above the main "download" link to redirect people.

I also added a README to the downloads which now points to GitHub and the installation instructions on cantera.org. I couldn't figure out any way to get rid of the big green "Download" button on the landing page, but this will help if anyone gets as far as the download index page.

As far as the CERFACS fork, I don't think linking to some inaccessible version of the code is very helpful. It would just serve to make those pages, with their documentation for versions of Cantera that are either long-deprecated or not publicly available more prominent.
